CHANGELOG.md

```diff
@@ -2,18 +2,16 @@
 
 ## [0.2.0](https://www.github.com/yandex-cloud/react-data-table/compare/v0.1.0...v0.2.0) (2022-01-11)
 
-
 ### ⚠ BREAKING CHANGES
 
-* theme previously known as 'common' is now called 'yandex-cloud' and is a default one, theme previously known as 'internal' is now called 'legacy'.
+- theme previously known as 'common' is now called 'yandex-cloud' and is a default one, theme previously known as 'internal' is now called 'legacy'.
```

Comment:
I strongly recommend to omit CHANGELOG.md from prettier checks because it is auto-generated by github action, otherwise, it will always be inconsistent.